draft-private-access-tokens.md

```diff
@@ -1211,6 +1211,10 @@ this 'single issuer/mediator' fashion reduces the privacy promises to those of t
 desirable for a redemption flow that is limited to specific issuers and mediators,
 but should be avoided where hiding origins from the mediator is desirable.
 
+A Mediator and Origin can collude to reveal a client's identity through timestamp correlation. The
+timing of a request to an Origin and subsequent token issuance to a Mediator can reveal the Client
```

Comment:
I think we should focus on correlating identity and access... it doesn't directly reveal just the client to the origin, it allows the origin and mediator to compare notes.